recipes/ruff/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "ruff" %}
+{% set version = "0.0.137" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0d0c7c8924cb5fee9daaf8f6f3f7d49e288df7b89bb619a645eb1f9208737285
+
+build:
+  number: 0
+  # the distributed binaries are already relocatable
+  binary_relocation: false
+  missing_dso_whitelist:
+    - /usr/lib/libresolv.9.dylib  # [osx]
+
+requirements:
+  build:
+    - posix  # [win]
+    # - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - clang
+    - llvm
+    - rust
+    - llvmdev
+    - llvm-tools
+    - libclang
+    - maturin
+    - toml
+    - pip
+  host:
+    - python
+    - pip
```

This mixes host and build dependencies. Probably the following should work:
```suggestion
    - {{ compiler('c') }}
    - {{ compiler('cxx') }}
    - {{ compiler('rust') }}
  host:
    - python
    - pip
    - maturin
    - toml
```